production-marathon/README.md

```diff
@@ -0,0 +1,58 @@
+# Production Marathon Deployment
+
+The Production Marathon deployment supports the deployment of the CAF Job Service on Mesos/Marathon. This folder contains the marathon environment and template files that are required to deploy the service application.
+
+## Service Configuration
+
+### Marathon Template
+The `marathon.json.b` template file describes the marathon deployment information required for Job Service. The template file uses property substitution to get values for configurable properties **required** for service deployment. These properties are configured in the marathon environment file `marathon.env`.
+
+### Marathon Environment
+The `marathon.env` file supports configurable property settings necessary for service deployment. These include:
+
+- `DOCKER_REGISTRY`: This setting configures the docker repository that the Job Service image will be pulled from. 
+
+- `JOB_SERVICE_PORT`: This configures the port that the Job Service listens on. 
+
+- `JOB_TRACKING_SERVICE_PORT`: This configures the port that the Job Tracking worker listens on
+
+- `POSTGRES_DB_HOSTNAME`: This configures the host name for the Postgres database.
+
+- `POSTGRES_DB_PORT`: This configures the port for the Postgres database.
+
+- `POSTGRES_JOB_SERVICE_DB_USER`: The username for the Postgres database.
+
+- `POSTGRES_JOB_SERVICE_DB_PASSWORD`: The password for the Postgres database.
+
+- `CAF_RABBITMQ_HOST`: This configures the host for RabbitMQ.
+
+- `CAF_RABBITMQ_PORT`: This configures the port for RabbitMQ.
+
+- `CAF_RABBITMQ_USERNAME`: This configures the username for RabbitMQ.
+
+- `CAF_RABBITMQ_PASSWORD`: This configures the password for RabbitMQ.
+
+
+### Additional Marathon Configuration
+The `marathon.json.b` deployment template file specifies default values for a number of additional settings which you may choose to modify directly for your custom deployment. These include:
+
+##### Application CPU, Memory and Instances
+
+- `cpus` : This setting can be used to configure the amount of CPU of each Job Service and Job Tracking container. This does not have to be a whole number. 
+	- **Default Values:**
```

Comment:
I pulled the previous values from the docker-compose file in job-service deploy. However speaking with Greg and in keeping with consistency across the deployment stages I've upped the minimum specs to what he has in his production compose file (I'll add the commit in after this comment). He deem's these sufficient for minimum specs